llvm/tools/llvm-exegesis/lib/SerialSnippetGenerator.cpp

```diff
@@ -54,12 +54,6 @@ computeAliasingInstructions(const LLVMState &State, const Instruction *Instr,
       continue;
     const Instruction &OtherInstr = State.getIC().getInstr(OtherOpcode);
     const MCInstrDesc &OtherInstrDesc = OtherInstr.Description;
```

Comment:
This line should be removed, the variable is unused.

Reply:
Thanks for noticing! Removed